gwadoc/doc_en.py

```diff
@@ -2471,123 +2448,123 @@
 ### Relation Simple Aspect IP
 
 relations.simple_aspect_ip.name.en = "Simple Aspect (imp to per)"
-relations.simple_aspect_ip.df.en = "A concept which is linked to another through a change from imperfective to perfective aspect"
+relations.simple_aspect_ip.df.en = "a concept which is linked to a given concept through a change from imperfective to perfective aspect"
 relations.simple_aspect_ip.ex.en = ""
 
 ### Relation Simple Aspect PI
 
 relations.simple_aspect_pi.name.en = "Simple Aspect (per to imp)"
-relations.simple_aspect_pi.df.en = "A concept which is linked to another through a change from perfective to imperfective aspect"
+relations.simple_aspect_pi.df.en = "a concept which is linked to a given concept through a change from perfective to imperfective aspect"
 relations.simple_aspect_pi.ex.en = ""
 
 
 ### Relation Secondary Aspect IP
 
 relations.secondary_aspect_ip.name.en = "Secondary Aspect (ip)"
-relations.secondary_aspect_ip.df.en = "A concept which is linked to another through a change in aspect (ip)"
+relations.secondary_aspect_ip.df.en = "a concept which is linked to a given concept through a change in aspect (imperfective to perfective)"
```

Comment:
Is this different from "Simple Aspect (imp to per)"?

Reply:
I am not sure what the difference of these relations are. It seems to be something that only really makes sense for Polish and other Slavic languages, so we may have to ask Maciej